packages/wrapped-keys/src/lib/lit-actions-client/utils.ts

```diff
@@ -48,3 +51,12 @@ export function postLitActionValidation(
 export function getLitActionCid(network: Network, actionType: LitActionType) {
   return LIT_ACTION_CID_REPOSITORY[actionType][network];
 }
+
+export function getLitActionCode(
+  network: Network,
+  actionType: LitActionType
+): string | undefined {
+  const litActionCode = LIT_ACTION_BUNDLED_CODE_PATH[actionType][network];
```

Comment:
should we check if `litActionCode` is defined before returning?

Reply:
And throw if it is not? It is kind of expected that you can get an falsy value (empty string). That is even normal operation when the bundle does not include them and you have to use the CID
This way we can define just a subset of bundled LAs because the querying is the same than for CID. But it also makes sense to throw now

Let me know if this is what you were thinking or I misunderstood

Reply:
Yes this is my thought process. I think throwing is the best option since returning `undefined` will probably cause undefined behaviors from the caller perspective.

Reply:
Updated. Also added a check for having one or the other. lit-node-client does this, but we can already validate it here